src/plugins/data_source_management/public/components/data_source_menu/data_source_menu.tsx

```diff
@@ -80,7 +81,6 @@ export function DataSourceMenu<T>(props: DataSourceMenuProps<T>): ReactElement |
         hideLocalCluster={hideLocalCluster || false}
         fullWidth={fullWidth}
         uiSettings={uiSettings}
-        application={application}
```

Comment:
why removing application here?

Reply:
can you check the main branch https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/data_source_management/public/components/data_source_menu/data_source_menu.tsx and make sure we are not deleting existing implementations? cc @zhongnansu